android/src/main/java/com/reactnativeandroidwidget/builder/widget/TextWidget.java

```diff
@@ -129,4 +135,22 @@ private int getFontSizeUnit() {
         }
         return COMPLEX_UNIT_SP;
     }
+
+     private void setDecorationLine(String decoration) {
+        switch (decoration) {
+            case "underline":
+                view.setPaintFlags(view.getPaintFlags() | Paint.UNDERLINE_TEXT_FLAG);
+                break;
+            case "line-through":
+                view.setPaintFlags(view.getPaintFlags() | Paint.STRIKE_THRU_TEXT_FLAG);
+                break;
+            case "underline line-through":
+                view.setPaintFlags(view.getPaintFlags() | Paint.UNDERLINE_TEXT_FLAG | Paint.STRIKE_THRU_TEXT_FLAG);
+                break;
+            case "none":
```

Comment:
Looks better.

Please remove the `none` case, as well as the `default` case.

Default for text is no text decoration, there is no need to do anything in this case.